README.md

```diff
@@ -66,7 +72,7 @@ Use a VISC Report:
 
 ``` r
 use_visc_report(
-  report_name = "VDCnnn_BAMA_PTreport_interim_blinded", # the name of the report file
```

Comment:
Hmm ok, I think this is my fault for editing README.md in a previous PR rather than README.Rmd. The "VDCnnn_BAMA_PTreport_interim_blinded" syntax is the correct one, though. Since we're catching this now, can you update README.Rmd with this syntax and recompile README.md?

Reply:
Done!